lib/src/chart/base/axis_chart/axis_chart_painter.dart

```diff
@@ -76,6 +80,25 @@ abstract class AxisChartPainter<D extends AxisChartData> extends BaseChartPainte
     }
   }
 
+  /// This function draws a colored background behind the chart.
+  void drawBackground(Canvas canvas, Size viewSize) {
+    if (data.backgroundColor == null) {
```

Comment:
If we have default color, there is no need to null check, and it's okay we throw a error when user passed null.

Reply:
Ok, but on the other hand, this will cause an unnecessary rendering. And transparent/translucent rendering is often more expensive over opaque rendering. I'm not sure if this is an issue with flutter/skia.